udi-core/src/main/java/no/nav/registre/core/provider/rs/ControllerAdvice.java

```diff
@@ -38,8 +38,8 @@ public Map constraintHandler(ConstraintViolationException exception, HttpServlet
         var annotation = Optional.ofNullable(AnnotationUtils.findAnnotation(ex.getClass(), ResponseStatus.class));
         var status = annotation.map(ResponseStatus::code).orElse(HttpStatus.INTERNAL_SERVER_ERROR);
         if (annotation.isEmpty()) {
-            log.error("Internal server error", ex);
-            return ResponseEntity.status(status).body(getValues(ex, request, "Internal server error", status));
+            log.error(HttpStatus.INTERNAL_SERVER_ERROR.name(), ex);
```

Comment:
Log exception message i stedet for navnet på exception.

Reply:
Enig, fikset.